ext/util/math.c

```diff
@@ -434,6 +434,13 @@ php_driver_format_integer(mpz_t number, char **out, int *out_len)
   tmp = (char*) emalloc((len + 1) * sizeof(char));
   mpz_get_str(tmp, 10, number);
 
+  // mpz_sizeinbase returns the length of the number, but it can be 1 more than the actual length.
```

Comment:
Yeah, I saw that, too, but I thought our existing code was trying to avoid the strlen call.